app/uk/gov/hmrc/cdsreimbursementclaimfrontend/models/CompleteClaim.scala

```diff
@@ -74,7 +73,7 @@ object CompleteClaim {
     maybeDuplicateDisplayDeclaration: Option[DisplayDeclaration],
     importerEoriNumber: Option[CompleteImporterEoriNumberAnswer],
     declarantEoriNumber: Option[CompleteDeclarantEoriNumberAnswer],
-    completeClaimsAnswer: CompleteClaimsAnswer
+    completeClaimsAnswer: ClaimsAnswer
```

Comment:
Lets rename `completeClaimsAnswer` to `claimsAnswer`

Reply:
Done